src/Entity/Message.php

```diff
@@ -409,4 +410,16 @@ public function setLanguage($language) {
     $this->language = $language;
   }
 
+  /**
+   * Default value callback for 'uid' base field definition.
+   *
+   * @see ::baseFieldDefinitions()
```

Comment:
I think it's fine to copy this from node, and don't bother mentioning it. There isn't too much going on here.